llama_index/schema.py

```diff
@@ -380,6 +380,17 @@ def to_langchain_format(self) -> LCDocument:
     def from_langchain_format(cls, doc: LCDocument) -> "Document":
         """Convert struct from LangChain document format."""
         return cls(text=doc.page_content, metadata=doc.metadata)
+    
+    @classmethod
+    def for_test() -> "Document":
```

Comment:
Is there a reason to prefer doing this over creating a pytest fixture?

Reply:
Hmm, not really, but it's not just for unit tests, it's also just convenience when you wanna test something out.

```python
from llama_index import Document, VectorStoreIndex

index = VectorStoreIndex.from_documents([Document.for_test()])
```

Reply:
Actually, maybe it should be a more legit document in this case.

Reply:
Something that actually has semantic content, like the paul graham essay

Reply:
More for testing semantically-relevant features. E.g. testing the GraphIndex

Reply:
Changed the method to `example` to distinguish the use case from test.